client/my-sites/patterns/components/pattern-preview/index.tsx

```diff
@@ -83,6 +96,23 @@ function PatternPreviewFragment( {
 		return null;
 	}
 
+	const recordOpenModalEvent = ( tracksEventName: string ) => {
+		recordTracksEvent( tracksEventName, {
+			name: pattern?.name,
+			category,
+			type: getTracksPatternType( patternTypeFilter ),
+			view,
```

Comment:
Only in thsi case we track `view`, but don't do it for `closed` / `sign up` \ `log in`. I suppose that we should always track the same parameters or no?

Reply:
Good point. If I had to theorize, I'd guess that `view` was included in this one because it could be valuable to know if grid vs list had an impact on how likely a user is to click the Get Access button. For the close/login/signup flows, the modal is already opened, so the `view` is probably less important.

Unless there's a downside to including it in everything I can do so, and wrap them in into a single handler accepting only an event name.